modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceEntryLocalServiceImpl.java

```diff
@@ -750,6 +718,38 @@ public CommercePriceEntry updateExternalReferenceCode(
 		return commercePriceEntryPersistence.update(commercePriceEntry);
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommercePriceEntry updateHasTierPrice(
+			long commercePriceEntryId, boolean hasTierPrice)
+		throws PortalException {
+
+		CommercePriceEntry commercePriceEntry =
+			commercePriceEntryPersistence.findByPrimaryKey(
+				commercePriceEntryId);
+
+		commercePriceEntry.setBulkPricing(true);
+		commercePriceEntry.setHasTierPrice(hasTierPrice);
+
+		return commercePriceEntryPersistence.update(commercePriceEntry);
+	}
+
+	@Override
+	public CommercePriceEntry updateHasTierPrice(
+			long commercePriceEntryId, boolean hasTierPrice,
+			boolean bulkPricing)
```

Comment:
can we sort these to `long commercePriceEntryId, boolean bulkPricing, boolean hasTierPrice` ?